recipe/meta.yaml

```diff
@@ -24,11 +25,12 @@ requirements:
     - {{ stdlib('m2w64_c') }}  # [win]
   host:
     - python
+    - pip
     - numpy
     - meson-python
-    - setuptools <74
   run:
     - python
+    - {{ pin_compatible('numpy') }}
```

Comment:
If this works with numpy >=2 we no longer need this. You can declare just numpy here instead.